File: common/src/main/java/eu/darken/porter/common/UserServiceLaunch.java
package eu.darken.porter.common;

/**
 * Porter-only launch validation for user services; independent of the Shizuku API.
 *
 * <p>Lives here rather than in {@code :server} because the starter asks the question and
 * {@code :server} already depends on {@code :starter}.
 */
public final class UserServiceLaunch {
    public static final int TRANSACTION = 10007;
    private UserServiceLaunch() {}
}

File: common/src/main/java/eu/darken/porter/common/UserServiceLaunch.kt
package eu.darken.porter.common

/**
 * Porter-only launch validation for user services; independent of the Shizuku API.
 *
 * Lives here rather than in `:server` because the starter asks the question and `:server`
 * already depends on `:starter`.
 */
object UserServiceLaunch {
    const val TRANSACTION = 10007
}
